src/runtime/runtime_internal.h

```diff
@@ -49,6 +49,9 @@ typedef int32_t intptr_t;
 #define STDOUT_FILENO 1
 #define STDERR_FILENO 2
 
+#define O_RDONLY 0
```

Comment:
Please make these enums or constexprs if at all possible; putting new #defines into a .h file should really be be avoided unless absolutely unavoidable